meeting.js

```diff
@@ -0,0 +1,346 @@
+// Prerequisite data SIP and Access Token (Hard Coded)
+const myAccessToken = "MWJiZWQ3MzctNTllZS00YjMxLTg4MTQtNGNhYjIwNjBkZGI3Njk5MDBmY2UtOTUx_PF84_1eb65fdf-9643-417f-9974-ad72cae0e10f";

```

Comment:
Let's not commit any access token values going forward. These are public repositories and having an access token like that could sabotage your account